lib/engine/game/g_1829/game.rb

```diff
@@ -138,9 +137,22 @@ def all_potential_upgrades(tile, tile_manifest: false, selected_company: nil)
         end
 
         def setup
-          corporations.each do |i|
-            @stock_market.set_par(i, @stock_market.par_prices.find { |p| p.price == PARPRICE[@corporations.index(i)] })
-            i.ipoed = true
+          par_prices = {
```

Comment:
make this a constant and put it on line 38

Reply:
i' skiped the dictionary, because i don't know the way to access the second value.

Reply:
dictionary.each do |key, value|